src/main/resources/application.yml

```diff
@@ -1,4 +1,9 @@
 spring:
+  security:
```

Comment:
UserSpecification - Repeated code pattern for enum filtering violates DRY principle Extract the enum filtering logic into a private helper method to eliminate duplication and improve maintainability.